proteuslib/unit_models/reverse_osmosis_1D.py

```diff
@@ -563,49 +1029,118 @@ def initialize(blk,
         solve_log = idaeslog.getSolveLogger(blk.name, outlvl, tag="unit")
 
         # Create solver
+        if optarg is None:
+            optarg = {'nlp_scaling_method': 'user-scaling'}
+
         opt = get_solver(solver, optarg)
 
-        init_log.info('Starting initialization')
+        init_log.info('Starting Initialization Step 1: initialize blocks.')
```

Comment:
The 0D-RO unit model puts most of its initialization output in `info_high`, except to report success.

Reply:
I changed this because the messages were not being displayed when I set `info_high`.

Reply:
This snippet `ro_unit.initialize(outlvl=ideaslog.INFO_HIGH)` gives be the `info_high` initialization logs on the 0D-RO model.

Either way is perfectly valid; just pointing out an inconsistency.